subsys/bluetooth/host/Kconfig

```diff
@@ -173,6 +173,16 @@ config BT_SETTINGS
 	  instead of each subsystem doing it independently.
 
 if BT_SETTINGS
+config BT_SETTINGS_CCC_LOAD_ON_DEMAND
+	bool "Load CCC values from settings when peer connects"
+	default y
+	help
+	  Load Client Configuration Characteristic value right after bonded
+	  device connects.
+	  By default, CCC is only loaded when calling settings_load.
+	  Choosing this option is will save memory as the CCC values will only
```

Comment:
Since it it now default to y I guess we should explain the negative effect when disabling it (which would increase memory usage), not the other around.

Reply:
Will update.